src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -25,6 +25,7 @@ class ModuleExecutor(
                 val moduleName = response.first
                 response.second.fold(
                     {
+                        @Suppress("detekt:OptionalWhenBraces")
                         when (it) {
                             is OperationNotNeededModuleResponse -> {
                                 if (config[Arisa.Debug.logOperationNotNeeded]) {
```

Comment:
the second one looks fine to me. if detekt accepts that, i think that'd be better than suppressing.

Reply:
Hmm, that `OptionalWhenBraces` rule is actually deprecated (see [documentation](https://detekt.dev/docs/rules/style#optionalwhenbraces)), and not enabled by default (we just enable all rules in the build script).

Maybe it is also a bug that this rule now reports this issue, because at least the release notes don't seem to mention anything about this. And on the other hand while the second variant (without braces) seems to be accepted, I am bit surprised about that because I would expect that [`BracesOnWhenStatements`](https://detekt.dev/docs/rules/style#bracesonwhenstatements) with its default `multiLine=consistent` should prevent that.

Reply:
Maybe it's worth thinking about making the formatting rules less strict then, by not requiring literally every rule to pass? I'd assume there are some presets that are less pedantic.

Reply:
There is https://github.com/detekt/detekt/blob/main/detekt-core/src/main/resources/default-detekt-config.yml (but that also disables some non-style rules); not sure if there are any other default configs.

Maybe we can also just wait and see if the behavior changes in the version released after 1.23.1, whenever that might be.